src/wp-includes/block-bindings/sources/post-meta.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Add the post_meta source to the Block Bindings API.
+ *
+ * @since 6.5.0
+ * @package WordPress
+ */
+function post_meta_source_callback( $source_attrs ) {
+	// Use the postId attribute if available
+	if ( isset( $source_attrs['postId'] ) ) {
+		$post_id = $source_attrs['postId'];
+	} else {
+		// $block_instance->context['postId'] is not available in the Image block.
+		$post_id = get_the_ID();
+	}
+
+	return get_post_meta( $post_id, $source_attrs['value'], true );
+}
+
+wp_block_bindings_register_source(
+	'core/post_meta',
```

Comment:
Let's follow the same naming pattern as for block types:

```suggestion
	'core/post-meta',
```

Reply:
Addressed in https://github.com/WordPress/wordpress-develop/pull/5888/commits/45a96f8d74f69bfdedd1a10d208e565c1fbf7017